vulcan/hebe/data/_lesson.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+from typing import Union, AsyncIterator, List
+import datetime
+from related import (
+    immutable,
+    IntegerField,
+    StringField,
+    ChildField,
+    BooleanField,
+    SequenceField,
+    TimeField,
+)
+
+from .._api_helper import FilterType
+from .._endpoints import DATA_TIMETABLE
+from ..model import Serializable, DateTime, Teacher, Subject, TeamClass, TeamVirtual
+from ..._utils import TIME_FORMAT_H_M, sort_and_filter_dates
+
+
+@immutable
+class LessonTime(Serializable):
+    """Lesson time (start-end range)
+
+    :var int ~.id: Lesson time ID
+    :var `datetime.time` ~.from_: Lesson start time
+    :var `datetime.time` ~.to: Lesson end time
+    :var str ~.displayed_time: lesson's displayed time
+    :var int ~.position: Lesson position
+    """
+
+    id: int = IntegerField(key="Id")
+    from_: TimeField = TimeField(key="Start", formatter=TIME_FORMAT_H_M)
+    to: TimeField = TimeField(key="End", formatter=TIME_FORMAT_H_M)
+    displayed_time: str = StringField(key="Display")
+    position: int = IntegerField(key="Position")
+
+
+@immutable
+class LessonRoom(Serializable):
+    """Lesson room
+
+    :var int ~.id: Lesson room ID
+    :var str ~.code: Classroom code
+    """
+
+    id: int = IntegerField(key="Id")
+    code: str = StringField(key="Code")
+
+
+@immutable
+class LessonGroup(Serializable):
```

Comment:
Use the TeamVirtual class (hebe.model.TeamVirtual) instead for consistency with other modules.